app/src/main/res/values/strings.xml

```diff
@@ -94,4 +94,18 @@
     <!-- debug log - title -->
     <string name="debug_log_title">Log</string>
 
+<!--    Transaction Details-->
+    <string name="payment_received">Payment Received</string>
```

Comment:
Please id your string resources just like the way you id your layout components.

[activity_or_fragment_name]_[string_resource_name]

For this instance it it tx_details_payment_received.

If you think the resource is something more general and can be used by other views, then you can move it around the top of the file and prefix it with common, such as:

common_payment_received